apps/website/src/pages/show-and-tell/index.tsx

```diff
@@ -391,6 +401,9 @@ const ShowAndTellIndexPage: NextPage<ShowAndTellPageProps> = ({
           onKeyDown={handleArrowKeys}
           tabIndex={-1}
         >
+          <h1 className="text-center text-3xl italic">
+            {totalPostsCount} posts by {usersCount} users
```

Comment:
```suggestion
            {totalPostsCount} posts by {usersCount} community members
```
I think it'd be nicer to refer to users as community members here